compiler/testData/diagnostics/tests/deprecated/javaDeprecated.fir.kt

```diff
@@ -11,6 +11,6 @@ public class A {
 
 // FILE: B.kt
 
-class B(private <!INAPPLICABLE_CANDIDATE, INAPPLICABLE_CANDIDATE!>@property:Deprecated<!> val foo: String) : A() {
+class B(private @property:Deprecated val foo: String) : A() {
```

Comment:
Did you investigate this case? Is it safe to remove those errors?

Reply:
This is safe since the more detailed error `NO_VALUE_FOR_PARAMETER` is disabled for this test case on top of this fiel